### PR TITLE
Fix reserve action button

### PR DESCRIPTION
**Pull request by samaradel**
### Description

Fix the reserve action button.

### Changes

- Reduce check time of Un/Reserved buttons
- Check the twin ID of the rented node when the component is mounted
-  Reset values after reserve / unreserve process

### Related Issues

- https://github.com/threefoldtech/tfgrid-sdk-ts/issues/3575

### Tested Scenarios

- Navigate to any solution
- Switch on dedicated nodes
- Reserve any node and unreserve it

### Updated Scenarios
- Reserve from the node page and un-reserve any from the solution page.
- Reserve from a solution page, then un-reserve from the node page.

### Checklist

- [x] Tests included
- [x] Build pass
- [ ] Documentation
- [x] Code format and docstrings
- [ ] Screenshots/Video attached (needed for UI changes)

### CODE REVIEW

**Review by maayarosama — APPROVED** [review]

**Review by ramezsaeed — COMMENTED** [review]
- Try reserve a dedicated node from solution X, and un-reserve it from solution Y.
- Reserve from node page and un-reserve any from solution page.
- Reserve from a solution page, then un-reserve from node page.

File: packages/playground/src/dashboard/components/reserve_action_btn.vue
Comment (0oM4R) [inline packages/playground/src/dashboard/components/reserve_action_btn.vue]: i suggest changing this to an error, not info, as the process of unreserve is failed

Comment (0oM4R) [reply packages/playground/src/dashboard/components/reserve_action_btn.vue]: also small comment line before the request will make our life easier in the future :D, to mention that we are checking if there is contract on the rented node

**Review by 0oM4R — COMMENTED** [review]
I think we should add await on the time out to keep loading untill the val;ue got updated 

[Screencast from 2024-11-25 13-24-48.webm](https://github.com/user-attachments/assets/8260b064-b60d-4133-b62e-bfa0ec1bbc26)

**Comment by ramezsaeed** [comment]
Please confirm these scenarios had been tested; and add them to the PR body.
https://github.com/threefoldtech/tfgrid-sdk-ts/pull/3650#pullrequestreview-2456695150

**Comment by samaradel** [comment]
@ramezsaeed I'm waiting for https://github.com/threefoldtech/tfgrid-sdk-ts/pull/3587 as I can't see any dedicated nodes

**Comment by samaradel** [comment]
@ramezsaeed the scenarios are tested except for the first case which is blocked on https://github.com/threefoldtech/tfgrid-sdk-ts/pull/3587

**Comment by maayarosama** [comment]
> @ramezsaeed the scenarios are tested except for the first case which is blocked on #3587

All scenarios passed, except the first one as it's blocked on another issue, Once the its pr is merged will approve

**Pull request closed without merging**
